subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1842,7 +1888,6 @@ static void le_conn_complete(struct pdu_data *pdu_data, u16_t handle,
 	sep->supv_timeout = sys_cpu_to_le16(radio_cc->timeout);
 	sep->clock_accuracy = radio_cc->mca;
 
-	conn_count++;
```

Comment:
There's a redundant empty line left at the end of the function once you remove this.